### PR TITLE
Rebuild v1.9.0 with libarchive_static_for_cph 3.6.1

### PR DESCRIPTION
**Pull request by skupr-anaconda**
**Jira ticket:** [PKG-374](https://anaconda.atlassian.net/browse/PKG-374) (conda-package-handling-
1.9.0)

**The upstream data:**
Github releases:  https://github.com/conda/conda-package-handling/releases
License: https://github.com/conda/conda-package-handling/blob/master/LICENSE
Changelog: https://github.com/conda/conda-package-handling/blob/main/CHANGELOG.md
Requirements:
 * setup.cfg:  https://github.com/conda/conda-package-handling/blob/1.9.0/setup.cfg
 * setup.py:  https://github.com/conda/conda-package-handling/blob/1.9.0/setup.py

**_Actions:_**

1. Bump build number to `1`.
2. Remove the old `missing_dso_whitelist`, see the commit when it was added https://github.com/AnacondaRecipes/conda-package-handling-feedstock/commit/8b8ed22f6d5872769dd8bca291fb866d5825868d
3. Use `_libarchive_static_for_cph ==3.6.1`
4. Add missing `setuptools` and `wheel`
5. Add `license_url`
6. Add a `description`

**_Notes:_**
 * To address the [issue #5 ](https://github.com/AnacondaRecipes/conda-package-handling-feedstock/issues/5)

**Package's statistics**
<details>

 * Priority B | effort: easy | Category: miniconda | subcategory:  | pkg_type: python
 * Outdated platfroms: 
 * Latest version: 1.9.0
 * Popularity: 
    * 3 months downloads: 3581169.0
    * All time:  8862458 downloads -  -package-handling  1.9.0  Create and extract  packages of various formats  

</details>

**Links:**
<details>

* [The v1.9.0 upstream URL](https://github.com/conda/conda-package-handling/tree/1.9.0)
* [dev_url](https://github.com/conda/conda-package-handling)
* [doc_url](https://github.com/conda/conda-package-handling)
* [Bug tracker](https://github.com/conda/conda-package-handling/issues)
* [PyPi](https://pypi.org/project/conda-package-handling)
* [conda-forge recipe](https://github.com/conda-forge/conda-package-handling-feedstock/blob/master/recipe/meta.yaml)
* [Anaconda recipe feedstock](https://github.com/AnacondaRecipes/conda-package-handling-feedstock)
* [Update branch: _rebuild_1.9.0_b1_](https://github.com/AnacondaRecipes/conda-package-handling-feedstock/tree/rebuild_1.9.0_b1)
* [Diff between upstream conda-forge **main** and aggregate **feature** branch](https://github.com/conda-forge/conda-package-handling-feedstock/compare/main...AnacondaRecipes:rebuild_1.9.0_b1#files_bucket)
* [Diff between origin aggregate **master** and aggregate **feature** branch](https://github.com/AnacondaRecipes/conda-package-handling-feedstock/compare/master...AnacondaRecipes:rebuild_1.9.0_b1#files_bucket)
* [The last merged PRs](https://github.com/AnacondaRecipes/conda-package-handling-feedstock/pulls?q=is%3Apr+is%3Amerged+sort%3Aupdated-desc+)

</details>

**Anaconda Linter:**

<details>

The following problems have been found:

ERROR: /aggregate/conda-package-handling-feedstock/recipe/meta.yaml:30: missing_wheel: For pypi packages, wheel should be present in the host section
ERROR: /aggregate/conda-package-handling-feedstock/recipe/meta.yaml:48: missing_pip_check: For pypi packages, pip check should be present in the test commands
WARNING: /aggregate/conda-package-handling-feedstock/recipe/meta.yaml:64: missing_doc_source_url: The recipe is missing a doc_source_url
WARNING: /aggregate/conda-package-handling-feedstock/recipe/meta.yaml:64: missing_description: The recipe is missing a description
WARNING: /aggregate/conda-package-handling-feedstock/recipe/meta.yaml:64: missing_license_url: The recipe is missing a license_url
Errors were found

</details>

**Crender checks:**

<details>
1.8.1:
- In build section empty missing_dso_whitelist specified
- Missing wheel package in requirements
- Missing setuptools package in requirements
- No documentation url provided

</details>

**Other checks:**

<details>

1. - [x] https://github.com/conda/conda-package-handling/tree/1.9.0 exists
7. - [x] https://github.com/conda/conda-package-handling is present
8. - [ ] Check the pinnings
9. - [x] https://github.com/conda/conda-package-handling/blob/main/CHANGELOG.md exists
10. - [ ] Additional research
    https://github.com/conda/conda-package-handling/issues
    200
11. - [x] `dev_url` is present
    https://github.com/conda/conda-package-handling
12. - [ ] `doc_url` error:
    
    
13. - [ ] Verify that the `build_number` is correct
14. - [ ] Verify if the package needs `setuptools`
    
15. - [ ] Verify if the package needs `wheel`
    
16. - [ ] NO `pip` in test
    
17. - [ ] Verify the test section
18. - [ ] Verify if the package is `architecture specific` or `Noarch`
19. - [ ] Verify that private module are not mentioned on the recipe For example: (_private_module)
20.  - [ ] license_file doesn't exist!
21. - [x] License: BSD-3-Clause
    - [ ] License is `spdx` compliant
 * Check if the license identifier has correct name from the SPDX License List

| Identifier                           |
|:-------------------------------------|
| BSD-3-Clause                         |
| BSD-3-Clause-Attribution             |
| BSD-3-Clause-Clear                   |
| BSD-3-Clause-LBNL                    |
| BSD-3-Clause-Modification            |
| BSD-3-Clause-No-Military-License     |
| BSD-3-Clause-No-Nuclear-License      |
| BSD-3-Clause-No-Nuclear-License-2014 |
| BSD-3-Clause-No-Nuclear-Warranty     |
| BSD-3-Clause-Open-MPI                |

22. - [x] license_family BSD is present
</details>

**Updating the recipe:**

If the recipe needs additional modification the update branch can be modified.

```
git clone -b 1.9.0 git@github.com:AnacondaRecipes/conda-package-handling-feedstock.git
```

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,8 +22,10 @@ requirements:
     - python
     - pip
     - cython
-    - _libarchive_static_for_cph ==3.4.3
+    - _libarchive_static_for_cph ==3.6.1
```
Comment (dholth) [inline recipe/meta.yaml:25]: big jump

Comment (skupr-anaconda) [reply recipe/meta.yaml:25]: Hmm, it's only to address CVEs https://github.com/AnacondaRecipes/conda-package-handling-feedstock/issues/5 and https://github.com/AnacondaRecipes/_libarchive_static_for_cph-feedstock/pull/2#issue-1423733661. They are critical

Comment (dholth) [reply recipe/meta.yaml:25]: It's good and it's too bad we haven't updated it more frequently.

**Review by cbouss — APPROVED** [review]
LGTM

**Review by marcoesters — APPROVED** [review]
LGTM

**Review by varlackc — APPROVED** [review]
LGTM

**Pull request merged**